source/contracts/reporting/Market.sol

```diff
@@ -273,12 +281,14 @@ contract Market is DelegationTarget, ITyped, Initializable, Ownable, IMarket {
         return _crowdsourcer;
     }
 
-    function migrateThroughOneFork() public returns (bool) {
+    function migrateThroughOneFork(uint256[] _payoutNumerators, bool _invalid, string _description) public returns (bool) {
```

Comment:
These arguments are ignored when migrating a market that has not ended?

Reply:
Yea

Reply:
Worth a comment? Optional required parameters are a bit unusual